UKHomeOffice/sgmr-service
# NMSW-762 Force value of `ais` field

### AC / Description of changes

Currently

- AIS has been removed from the UI
- AIS I believe has been removed from required fields in API

However

- AIS is returned in the data from a GET call

The issue

When doing a cancel/replace process for voyage the

- FE makes a GET call to get the existing voyage data
- GET returns the data including `ais: null`
- FE makes a POST to get a new voyage id
- FE PATCHes all the voyage data including any updates, and as part of this was passing `ais: null`
- A temporary fix is going in where FE will format the data from the GET to force `ais: 'not set'`

 
The real fix will be to remove ais from the API GET call so there is no need to reformat it's value before PATCHing voyage data for a resubmit.  This will come later.


### To Test

- create a voyage
- submit the voyage
- go back into the voyage and click one of the change options
- change something
- click save and continue
- > you should be returned to the CYA page and be able to submit
(the bug caused the save and continue to error) 

### Notes

Also fixed a test that was failing because the calculated date was 30th, and a time value was 30 so test was looking for one instance of `30` on the screen and finding two.

---

**Remember to merge to feature branches not `master`.**
